app/helpers/apa_helper.rb

```diff
@@ -1,15 +1,27 @@
 # frozen_string_literal: true
 
 module ApaHelper
-  def apa_citation(article)
+  def article_citation(article)
     title = article.title.capitalize_turkish
 
-    "#{authors_in_apa_format(article)}. (#{article.year}). #{title}. #{article.journal}, \
+    "#{authors_in_apa_format(article.authors)}. (#{article.year}). #{title}. #{article.journal}, \
     #{article.volume}(#{article.issue}), #{article.first_page}-#{article.last_page}"
   end
 
-  def authors_in_apa_format(article)
-    article.authors.split(',').map do |author|
+  def book_citation(book)
```

Comment:
```ruby
<<~TEXT.strip.squish
  #{authors_in_apa_format(book.authors)}.
  (#{book.year}).
  #{book.name.capitalize_turkish}.
  #{[book.city, book.publisher].compact.join(': ')}
TEXT
```

Not: `#{[book.city]}: #{book.publisher}` yerine `#{[book.city, book.publisher].compact.join(': ')}` kullandım  çünkü eğer kitabın `city`'si yoksa `#{[book.city]}: #{book.publisher}`  ` : OMU Yayın Evi` şeklinde görüntüleniyordu. Bu durumda `:` görüntülenmemeli